services/app-api/putPhoneNumber.test.js

```diff
@@ -1,8 +1,103 @@
-import {main} from "./putPhoneNumber";
+import { main } from "./putPhoneNumber";
 
-it('Get Stub', async () => {
+import { RESPONSE_CODE } from "cmscommonlib";
+import dynamoDb from "./libs/dynamodb-lib";
 
-    const response =  main( {"source": "serverless-plugin-warmup"}, "foo")
-    expect(response).toBeInstanceOf(Promise)
+jest.mock("./libs/dynamodb-lib");
 
-})
+dynamoDb.update.mockImplementation(() => {
+  return true;
+});
+
+it("validation of input", async () => {
+  const mockObject = {
+    statusCode: 200,
+    body: {},
+    headers: {
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Credentials": true,
+    },
+  };
+
+  const expectedResponse = {
+    id: "statesubmitteractive@cms.hhs.local",
+    statusCode: 200,
+    body: RESPONSE_CODE.VALIDATION_ERROR,
+    headers: {
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Credentials": true,
+    },
+  };
+
+  await expect(main(mockObject))
+    .resolves.toStrictEqual(expectedResponse)
+    .catch((e) => {
+      RESPONSE_CODE.VALIDATION_ERROR, e;
```

Comment:
I think we want to console.log all testing errors, so please make this catch block the same as the final one: 
    .catch((e) => {
      console.log("error", e);
    });

Reply:
Done!